HW-04/hw4.py

```diff
@@ -0,0 +1,16 @@
+text = input('Enter your text: ')
+if text.isdigit():
+    text = int(text)
+    print(str(text)+' it`s numeric.')
+    print(type(text))
+    def is_even(text):
```

Comment:
It is good that you know how to use functions, but in this case, this function makes the code more complicated and less readable and adds one extra line of code. 
Functions are good when you do some atomic operation, or when you can reuse it, or to split big part of a program. But in this case, it is unjustified.